packages/ckeditor5-fullscreen/src/handlers/classiceditor.ts

```diff
@@ -37,6 +37,7 @@ export default class ClassicEditorHandler extends AbstractEditorHandler {
 			this.moveToFullscreen( editorUIView.toolbar.element!, 'toolbar' );
 			this.generatePresenceListElement();
 			this.generateDocumentOutlineElement();
+			this.registerFullscreenDialogPositionAdjustements();
```

Comment:
Let's move it to the abstract handler (IIRC the two above calls should also be there).